great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -541,31 +544,118 @@ def _display(
+
         dropdown_title_color: str = altair_theme["legend"]["titleColor"]
-        dropdown_title_font: str = altair_theme["font"]
-        dropdown_css: str = f"""
+        dropdown_font: str = altair_theme["font"]
+        dropdown_font_size: str = altair_theme["axis"]["titleFontSize"]
+        dropdown_text_color: str = altair_theme["axis"]["labelColor"]
+
+        # Altair does not have a way to format the dropdown input so the rendered CSS must be altered directly
+        altair_dropdown_css: str = f"""
             <style>
             span.vega-bind-name {{
                 color: {dropdown_title_color};
-                font-family: "{dropdown_title_font}";
+                font-family: {dropdown_font};
+                font-size: {dropdown_font_size}px;
                 font-weight: bold;
             }}
             form.vega-bindings {{
-              position: absolute;
-              left: 75px;
-              top: 30px;
+                color: {dropdown_text_color};
+                font-family: {dropdown_font};
+                font-size: {dropdown_font_size}px;
+                position: absolute;
+                left: 75px;
+                top: 28px;
             }}
             </style>
         """
-        display(HTML(dropdown_css))
+        display(HTML(altair_dropdown_css))
 
         # max rows for Altair charts is set to 5,000 without this
         alt.data_transformers.disable_max_rows()
 
-        chart: alt.Chart
-        for chart in themed_charts:
-            chart.display()
+        ipywidgets_dropdown_css: str = f"""
+            <style>
+            .widget-inline-hbox .widget-label {{
+                color: {dropdown_title_color};
+                font-family: {dropdown_font};
+                font-size: {dropdown_font_size}px;
+                font-weight: bold;
+            }}
+            .widget-dropdown > select {{
+                padding-right: 21px;
+                padding-left: 3px;
+                color: {dropdown_text_color};
+                font-family: {dropdown_font};
+                font-size: {dropdown_font_size}px;
+                height: 20px;
+                line-height: {dropdown_font_size}px;
+                background-size: 20px;
+                border-radius: 2px;
+            }}
+            </style>
+        """
+        display(HTML(ipywidgets_dropdown_css))
+
+        dropdown_selection: widgets.Dropdown = widgets.Dropdown(
+            options=chart_titles,
+            description="Select Plot: ",
+            style={"description_width": "initial"},
+            layout={"width": "max-content", "margin": "0px"},
+        )
+
+        # As of 19 July, 2022 there is a DeprecationWarning due to the latest ipywidgets' interaction with
+        # ipykernel versions > 6.0.0. Rather than add a version constraint to ipykernel, we suppress
+        # DeprecationWarnings produced by module ipywidgets.widgets.widget_output
+        warnings.filterwarnings(
```

Comment:
Do we have a backlog item to keep track of this?
Do you know if there's a timeline for when this deprecated feature might be removed?

Reply:
There is nothing else we can do (that I know of) to suppress this warning, other than constrain the version of `ipykernel` - this is an issue with `ipywidgets` unfortunately. I don't see any issues in their list of [570 GitHub issues](https://github.com/jupyter-widgets/ipywidgets/issues) that discuss this warning, so I'm not sure what the timeline looks like.

> DeprecationWarning: Kernel._parent_header is deprecated in ipykernel 6. Use .get_parent()

Reply:
I meant should we create a backlog item to implement the change suggested by the deprecation warning?

Reply:
@Kilo59 there is no change that can be made on our side. The change needs to be made by the developers of `ipywidgets`.

Reply:
Oh lol, sorry, I just glazed over the difference between `ipywidgets` vs `ipykernel`